src/sensor/SerialInput.cpp

```diff
@@ -0,0 +1,19 @@
+#include "settings.h"
+#include "sensor/SerialInput.h"
+
+SerialInput::SerialInput(UartClass *serial) : Sensor<char>() {
```

Comment:
the child constructor implicitly calls the default parent constructor (unless you've overridden the default constructor for the parent, in which case you'll need to call it explicitly)